docker/oap-es7/docker-entrypoint.sh

```diff
@@ -294,202 +33,51 @@ validateVariables() {
     fi
 }
 
-generateApplicationYaml() {
-    # validate
-    [[ -z "$SW_CLUSTER" ]] && [[ -z "$SW_STORAGE" ]] && [[ -z "$SW_CONFIGURATION" ]] \
-        && [[ -z "$SW_TELEMETRY" ]] \
-        && { echo "Error: please specify \"SW_CLUSTER\" \"SW_STORAGE\" \"SW_CONFIGURATION\" \"SW_TELEMETRY\""; exit 1; }
-
-    validateVariables "SW_CLUSTER" "$SW_CLUSTER" "standalone zookeeper kubernetes consul etcd nacos"
-
-    validateVariables "SW_STORAGE" "$SW_STORAGE" "elasticsearch h2 mysql influxdb"
-
-    validateVariables "SW_CONFIGURATION" "$SW_CONFIGURATION" "none apollo nacos zookeeper"
-
-    validateVariables "SW_TELEMETRY" "$SW_TELEMETRY" "none prometheus so11y"
+echo "[Entrypoint] Apache SkyWalking Docker Image"
 
-    echo "# Generated by 'docker-entrypoint.sh'" > ${var_application_file}
-    #generate cluster
-    case ${SW_CLUSTER} in
-    standalone) generateClusterStandalone;;
-    zookeeper) generateClusterZookeeper;;
-    kubernetes) generateClusterK8s;;
-    consul) generateClusterConsul;;
-    etcd) generateClusterEtcd;;
-    nacos) generateClusterNacos;;
-    esac
+# validate
+[[ -z "$SW_CLUSTER" ]] || [[ -z "$SW_STORAGE" ]] || [[ -z "$SW_CONFIGURATION" ]] \
+    || [[ -z "$SW_TELEMETRY" ]] \
+    && { echo "Error: please specify \"SW_CLUSTER\" \"SW_STORAGE\" \"SW_CONFIGURATION\" \"SW_TELEMETRY\""; exit 1; }
```

Comment:
@hanahmily please review these, the original logic operators are `&&`, but according to the error message, I think they should be `||`, if I'm wrong, please correct me

Reply:
I think we should remove this assertation, leave it to oap config value checker.